operator/v1/types_console.go

```diff
@@ -42,6 +42,9 @@ type ConsoleSpec struct {
 	// If not specified, default route will be used.
 	// +optional
 	Route ConsoleConfigRoute `json:"route"`
+	// plugins defines a list of enabled console plugins.
+	// +optional
+	Plugins []string `json:"plugins,omitempty"`
```

Comment:
The operator config only has the plugin names, which reference a `ConsolePlugin` resource. The `ConsolePlugin` has the other details of the operator like the service.